common/src/test/java/org/apache/cassandra/sidecar/common/utils/TimeUtilsTest.java

```diff
@@ -0,0 +1,43 @@
+package org.apache.cassandra.sidecar.common.utils;
```

Comment:
license header is missing here

Reply:
Added license comment here and in one other place.